tests/yam/validate/validate_connectivity.pm

```diff
@@ -17,8 +17,8 @@ sub run {
     select_console 'root-console';
     my $ip_address_show = script_output("ip address show");
     record_info("ip address show", $ip_address_show);
-    my $connectivity = check_var('INST_COPY_NETWORK', '0') ? 'none' : 'full';
-    validate_script_output("nmcli networking connectivity", sub { m/\b$connectivity\b/ });
+    my $connectivity = check_var('INST_COPY_NETWORK', '0') ? 'none|unknown' : 'full';
+    validate_script_output("nmcli networking connectivity check", sub { m/\b$connectivity\b/ });
```

Comment:
for OEMDRV we can rely in the qemu setting that tell us that the sut is offline.